application/classes/Ushahidi/Validator/Post/Decimal.php

```diff
@@ -13,7 +13,7 @@ class Ushahidi_Validator_Post_Decimal extends Ushahidi_Validator_Post_ValueValid
 {
 	protected function validate($value)
 	{
-		if (!Valid::decimal($value)) {
+		if (!Valid::numeric($value)) {
```

Comment:
Switching to accept `1` was intentional since `1.0 == 1` so I don't think users care. We really just have decimal to accept any number at all while Integer only accepts integers.